README.md

```diff
@@ -1,88 +1,97 @@
+- Wait for TrueBlocks to build its cache. ([Why does this take so long?](#why-does-it-take-so-long-to-build-the-index))
+- [Query TrueBlocks for a list of transactions on your accounts](#using-the-api)
 
-| component | notes |
-|-----------|-------|
-| git       | Install the command line tool. [Instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)|
-| docker    | Install *docker* ([Instructions](https://docs.docker.com/engine/installation)).<br>- The community edition of docker (`docker-ce`) works fine.<br>- On Linux, grant permission to the current user to run docker (`sudo usermod -aG docker $USER`). |
-| docker-compose | Install [docker-compose](https://docs.docker.com/compose/install) |
 
-**Note**: Make sure that you are able to run `git`, `docker ps`, `docker-compose` without issue, and that you can do so without using the `sudo` command. ([trouble shooting]())
 
-### Important Note
+## Requirements
 
-Building and running the TrueBlocks docker image is taken care of by the dAppNode package manager. The following instructions are presented for those who are not running dAppNode. If you are running a dAppNode, you can simply install the package from the dAppNode package manager.
+- A running, syncing Parity Ethereum node with `--tracing on` enabled (see [Running A Node](#how-do-you-recommned-I-run-a-node))
 
-### Building the Docker Image
+- Time (see [Why Does It Take So Long To Build The Index?](#why-does-it-take-so-long-to-build-the-index))
 
-Make a local copy of the TrueBlocks docker repo on your machine:
+- git, docker, docker-compose (see [Prerequisites](#prerequisites))
 
-```
-$ git clone https://github.com/Great-Hill-Corporation/DAppNodePackage_trueblocks.git
-```
+## Installation
 
-Build the docker image:
+The following instructions are for building on your local machine. Running on DappNode? See our [Dappnode Instructions](#dappnode-instructions).
 
 ```
-$ docker-compose build
+git clone https://github.com/Great-Hill-Corporation/trueblocks-docker.git
+cd trueblocks-docker
+docker-compose build
 ```
-or, alternatively:
 
-```
-$ docker build --rm -f build/Dockerfile -t dnp_trueblocks:dev build 
-```
+## Before running
+
+Before you bring the TrueBlocks docker container up, take the time to configure the following options:
+- what port do you want to run the trueblocks api server on? Set this in docker-compose.yml. Right now, the ports line reads 80:80. If you want to change this from port 80 to, say, 8181, then change this to 8181:80.
+- What is your node's RPC endpoint? For some users, this is http://localhost:8545, but due to Docker's approach to networking, this is probably not the address you want to use (see [What is my RPC endpoint?](#what-is-my-rpc-endpoint)). Whatever the case, set your RPC endpoint in the trueblocks.public.dappnode.eth.env file to `RPC_PROVIDER=http://your-rpc-provider:port`.
+- Now you're ready to run trueblocks!
```

Comment:
The above section is probably going to need a lot more explanation. The 'hand waving' starts at "Whatever the case..." or "...this is probably not..." Re-read this from the perspective of someone who doesn't understand (like me) and you'll see that if http://locahost:8545 doesn't work, I am left with no recourse.

Reply:
I agree. I'm suggesting that the "a lot more explanation" part comes as a linked section ("What is my RPC endpoint?" I think I'll remove the details about localhost here. Too much cognitive load. They should resort entirely to the linked subsection.

Reply:
See #35